### ISSUE TITLE
About Nitrox

### ISSUE THREAD

**Issue by Alexejhero** [nkga/shinkai#2]
First of all, please excuse my bad english.
I read your blog post in which you wrote about Nitrox. The thing that stood out to me is that you simply copy pasted Nitrox code. 
For example, ShinkaiModel/Core/Log.cs is the same as NitroxModel/Core/Log.cs in the Nitrox repo (http://github.com/SubnauticaNitrox/Nitrox)
You just replaced Nitrox with Shinkai and moved on. You didnt even credit the original author. I find that to be very unethical. (Even the randomly generated UUIDs found in the .sln file are the same)
Even though Nitrox is no longer under development, you could just have made a pull reqest on the original mod, rather than just copying it. Who knows, someday, development may continue on it. 
If I were you, I would contact the original mod creator on Discord (you can find him in the Subnautica Modding Discord, link below) and discuss with him. If you really want to make a multiplayer mod, you should improve nitrox.
Thank you, and remember, ALWAYS credit work that isn't yours.
Subnautica Modding Discord: https://discord.gg/PMTwsKZ
Nitrox's creator youtube channel: https://www.youtube.com/channel/UClV2i0LTChvvmMxYUlTa52Q

**Comment by nkga** [comment]
This mod was based on Nitrox and is certainly no wholly original work. It shares a lot of the core even if the architecture ends up different at a high level. The blog post is a postmortem going over some of these differences.

While Shinkai was functional, the code quality is nowhere near the standards of the original project and why I felt that a real fork never offered much value. I released this more as a backup/reference of a project I developed to play with friends rather than something I would commit to long-term. It's also why I never advertised it anywhere with the only references to the project on GitHub, where developers may look (rather than players).

The name change is because it's not a direct fork for some of the reasons given above. In my experience, open source developers and licenses typically do not appreciate developers misrepresenting their software by reusing the original name. A good example of this is what happened with the Debian GNU/Linux distribution and Firefox/Iceweasel situation.
